src/Common/src/System/Drawing/ColorTable.cs

```diff
@@ -0,0 +1,87 @@
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace System.Drawing
+{
+    internal static class ColorTable
+    {
+        private static string ColorConstantsLock = "colorConstants";
+        private static Dictionary<string, Color> colorConstants;
+        private static string SystemColorConstantsLock = "systemColorConstants";
```

Comment:
`SystemColorConstantsLock  = new object();` ?
`SystemColorConstantsLock`  and `ColorConstantsLock` could also be combined as they are single use.